platform-sdk/docs/proposals/001-tss-rosters-design-proposal/proposal.md

```diff
@@ -0,0 +1,290 @@
+# TSS - Roster Proposal
```

Comment:
My comments from the previous revision still apply.

PS. It would be nice not to replace file names so as to allow rendering of diffs between revisions. It may be time consuming and unreasonable to re-read the entire old and new file just to try and catch (and probably miss some) changes.

Reply:
sorry about the renaming, just thought to bring it up to compliance with the Roster design template.

Reply:
Please see all the open conversations in this PR above. Here's the direct links:

https://github.com/hashgraph/hedera-services/pull/14045#discussion_r1653736679
https://github.com/hashgraph/hedera-services/pull/14045#discussion_r1653744591
https://github.com/hashgraph/hedera-services/pull/14045#discussion_r1653746653
https://github.com/hashgraph/hedera-services/pull/14045#discussion_r1653747683
https://github.com/hashgraph/hedera-services/pull/14045#discussion_r1653750604
https://github.com/hashgraph/hedera-services/pull/14045#discussion_r1653752584

Reply:
addressed these. If i miss any, please shout.